app/controllers/announcements_controller.rb

```diff
@@ -34,4 +36,28 @@ def index
   def scheduled_announcements
     Announcement.scheduled.order("scheduled_publication asc")
   end
+
+  def is_english_locale?
+    params[:locale].nil?
+  end
+
+  def redirect_to_news_and_communications
+    base_path = "#{Plek.new.website_root}/search/news-and-communications"
+    redirect_to("#{base_path}?#{news_and_communications_query_string}")
+  end
```

Comment:
I _think_ we don't need the fully path so it could become
```suggestion
    redirect_to("/search/news-and-communications?#{news_and_communications_query_string}")
  end
```
But shan't to demur if there is a good reason

Reply:
Redirecting to the root over the path is more specific, so there's less chance it can go wrong. It'll also work in development, when running Whitehall in isolation.